brilirs/benchmark.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+files=( "../benchmarks/ackermann.bril" "../benchmarks/binary-fmt.bril" "../benchmarks/check-primes.bril" \
+        "../benchmarks/collatz.bril" "../benchmarks/digital-root.bril" "../benchmarks/eight-queens.bril" \
+        "../benchmarks/euclid.bril" "../benchmarks/fib.bril" "../benchmarks/fizz-buzz.bril" \
+        "../benchmarks/gcd.bril" "../benchmarks/loopfact.bril" "../benchmarks/mat-mul.bril" \
+        "../benchmarks/orders.bril" "../benchmarks/perfect.bril" "../benchmarks/pythagorean_triple.bril" \
+        "../benchmarks/quadratic.bril" "../benchmarks/ray-sphere-intersection.bril" \
+        "../benchmarks/recfact.bril" "../benchmarks/sieve.bril" "../benchmarks/sqrt.bril" \
+        "../benchmarks/sum-bits.bril" "../benchmarks/sum-sq-diff.bril"
+        )
+jsons=( "../benchmarks/ackermann.json" "../benchmarks/binary-fmt.json" "../benchmarks/check-primes.json" \
+        "../benchmarks/collatz.json" "../benchmarks/digital-root.json" "../benchmarks/eight-queens.json" \
+        "../benchmarks/euclid.json" "../benchmarks/fib.json" "../benchmarks/fizz-buzz.json" \
+        "../benchmarks/gcd.json" "../benchmarks/loopfact.json" "../benchmarks/mat-mul.json" \
+        "../benchmarks/orders.json" "../benchmarks/perfect.json" "../benchmarks/pythagorean_triple.json" \
+        "../benchmarks/quadratic.json" "../benchmarks/ray-sphere-intersection.json" \
+        "../benchmarks/recfact.json" "../benchmarks/sieve.json" "../benchmarks/sqrt.json" \
+        "../benchmarks/sum-bits.json" "../benchmarks/sum-sq-diff.json"
+        )
+args=( "3 6" "128" "50" "7" "645634654" "8" "" "10" "101" "4 20" "8" "50 109658" "96 false" "496" "125" \
+        "-5 8 21" "" "8" "100" "" "42" "100")
```

Comment:
It is a little bit too bad that this script needs to re-list all the benchmarks and all their arguments instead of just using `../benchmarks/*.bril`. Maybe Brench could help with this—it could even wrap a call to Hyperfine? Not sure.

Reply:
I haven't really tried to use Brench yet but I will look into it. I was mostly going off of the blog post so I threw something together.

Reply:
The closest I've gotten is the following one-liner as a single stage pipeline for brench. `(export bril=$(bril2json </dev/stdin); export arg='3 6'; hyperfine -s basic --warmup 5 'echo $bril | brili -p $arg')` where I'm only trying to get the `Ackermann.bril` benchmark to run. I get a nice `ValueError: I/O operation on closed file.` when running Brench which I'm unsure of. It runs as expected on it's own(providing the ackermann file to stdin).

Reply:
Weird. Any chance it's a bash/sh difference?

Reply:
```sh
#!/bin/sh
ps -p $$

(export bril=$(bril2json </dev/stdin); export arg='3 6'; hyperfine -s basic --warmup 5 'echo $bril | brili -p $arg') < ../benchmarks/ackermann.bril
```
This appears to show the above line using /bin/sh and running without issue.